tests/report/common/tests.cpp

```diff
@@ -2,15 +2,292 @@
 // Licensed under the MIT License.
 
 #include "../common/tests.h"
+#include <openenclave/internal/crypto/cmac.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/report.h>
 #include <openenclave/internal/tests.h>
 
+#ifndef OE_BUILD_ENCLAVE
+#include "../../../host/sgx/sgxquoteprovider.h"
```

Comment:
shouldn't the sgx headers be inside some ifdef so it's not included on non-SGX machines?  Or even better, all the sgx-specific tests should be in a sgx-specific file.
(Also the use of the relative path "../../.." seems like bad practice)

Reply:
Opened #2205 for this.